commander/test/bootstrapping/commands/endpoint/invoke.spec.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import * as fs from 'fs-extra';
+import * as apiClient from '@liskhq/lisk-api-client';
+import * as Config from '@oclif/config';
+import { BaseIPCClientCommand } from '../../../../src/bootstrapping/commands/base_ipc_client';
```

Comment:
Not a show stopper but we can improve the importation of `BaseIPCClientCommand` (line 18) and `InvokeCommand` (line 20) to something like `import { BaseIPCClientCommand, InvokeCommand } from '../../../../src';` since they are both exported in the same `index.ts` file in the `src` dir.